integrations/spark/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -1,3 +1,3 @@
 Manifest-Version: 1.0 
-Premain-Class: marquez.spark.MarquezAgent
-Agent-Class: marquez.spark.MarquezAgent
\ No newline at end of file
+Premain-Class: openlineage.spark.agent.SparkAgent
+Agent-Class: openlineage.spark.agent.SparkAgent
```

Comment:
Do we need this file? We already define the manifest in [`spark/build.gradle`](https://github.com/MarquezProject/marquez/blob/main/integrations/spark/build.gradle#L165)

Reply:
I guess not, testing that

Reply:
we don't need it, gradle will create it for us.

Reply:
Nice 👍